### DISCUSSION TITLE
QInfiniteScroll can be improved by using Intersection Observer API

### DISCUSSION THREAD

**Discussion by bigradish** [quasarframework/quasar#13673] [Ideas / Proposals]
QInfiniteScroll currently checks the loading conditions tightly in the main thread. This may influence the performance of the app. If [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) is used, this issue can be resolved.
For using Intersection Observer API, the code of [v-lazy-image](https://github.com/alexjoverm/v-lazy-image/blob/master/v-lazy-image/index.js) can be referred.
Thank you very much.

**Comment by metalsadman** [comment]
Intersection component and v-intersection is using that api https://quasar.dev/vue-components/intersection
